src/bio/SConscript

```diff
@@ -16,7 +16,7 @@ def scons():
 
     bio = daos_build.library(denv, "bio", Glob('*.c'),
                              LIBS=['numa', 'spdk', 'smd'])
-    denv.Install('$PREFIX/lib/daos_srv', bio)
+    denv.Install('$PREFIX/lib64/daos_srv', bio)
```

Comment:
It won't let me comment on the scons_local change...was it intentional?   If not, please revert it.   I can't tell what changed but scons_local was updated recently in master.

Reply:
What's the best way to revert this scons_local commit accidentally made in solving conflict in master merge.
 Submodule scons_local updated 1 files
+21 −14		components/__init__.py

Do I have to revert all 2 commits + 1 merge in between, or is there a way to just update this __init__.py to master version?

Reply:
With help from Jeff, this is resolved.